wagtail/admin/templates/wagtailadmin/home/locked_pages.html

```diff
@@ -18,43 +18,45 @@
                 {% for page in locked_pages %}
                     <tr>
                         <td class="title" valign="top">
-                            <div class="title-wrapper">
-                                <a href="{% url 'wagtailadmin_pages:edit' page.id %}" title="{% trans 'Edit this page' %}">{{ page.get_admin_display_title }}</a>
+                            <div class="w-flex w-items-center w-justify-between w-gap-2">
```

Comment:
question: is there scope to bring these into one, more semantic class?

Reply:
I did consider it, but the current state of the HTML+CSS for the listings is very poor, and I wasn't sure if it's worth adding yet another class to `listing.scss` at the moment. The code badly needs a refactor to also align with our latest conventions (e.g. using `w-` prefix, not using `float`, etc.), so I chose not to add a new class for now. Wouldn't mind adding it if @thibaudcolas thinks it's a good idea though.